### PR TITLE
CSV Import Command

### PR DESCRIPTION
**Pull request by janabremer**
I wrote an console command to import my csv data, all details in the [user story](https://github.com/janabremer/capstone-project/issues/3)...

Looking forward to your reviews! :)

### CODE REVIEW

**Review by marianziegler — APPROVED** [review]
I have no comments or suggestions. Looks fine to me.

File: backend/config/services.yaml [hunk 1]
```diff
@@ -29,3 +29,10 @@ services:
 
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
+
+    console_command.csv_import:
+        class: App\Command\CsvImportCommand
+        arguments:
+            - "@doctrine.orm.entity_manager"
```
Comment (web-devil) [inline backend/config/services.yaml:36]: sehr gut!!

File: backend/src/Command/CsvImportCommand.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     * @param OutputInterface $output
+     *
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Attempting to import the feed...');
+
+        $csv = Reader::createFromPath('%kernel.root_dir%/../data/crop-products.csv', 'r');
```
Comment (web-devil) [inline backend/src/Command/CsvImportCommand.php]: please own service!

**Review by web-devil — CHANGES_REQUESTED** [review]

**Review by FabianAlleblas — COMMENTED** [review]
🙌 🚀

**Pull request merged**